apps/consumer-client/src/pages/examples/add-pcd.tsx

```diff
@@ -144,30 +147,43 @@ export default function Page(): JSX.Element {
         <br />
         <br />
         POD content to sign: <br />
-        <button
-          onClick={() => {
-            setPodContent(EXAMPLE_POD_CONTENT);
+        <textarea
```

Comment:
I unfortunately made some visual changes here in #1765 which are going to cause you conflicts.  Should be pretty easy to resolve so long as you know what I intended to change:
- Put the POD and GPC stuff in a separate ExampleContainer section.
- Change spelling of function names from "Pod" to "POD".
- Move the add POD button up above the GPC config (which you also did)
- Add a different "Test GPCs" folder config